bgs/bgs.go

```diff
@@ -281,6 +282,7 @@ type User struct {
 	Handle    string         `gorm:"uniqueIndex"`
 	Did       string         `gorm:"uniqueIndex"`
 	PDS       uint
+	Takedown  bool
```

Comment:
docs on this field? the others are pretty obvious. this one is less so.

Reply:
alternatively, we often use the pattern of having it be an optional timestamp (`TakedownAt`; weird noun-like, not verb-like) as a bit of a papertrail.